airflow/providers/vertica/hooks/vertica.py

```diff
@@ -46,5 +46,41 @@ def get_conn(self) -> connect:
         else:
             conn_config["port"] = int(conn.port)
 
+        boolOptions = ["connection_load_balance", "binary_transfer", "disable_copy_local", "request_complex_types", "use_prepared_statements"]
+        stdOptions = ["session_label", "backup_server_node", "kerberos_host_name", "kerberos_service_name", "log_path", "unicode_error", "workload", "ssl"]
+        conn_extra = conn.extra_dejson
```

Comment:
Any reason for going for a camel case instead of snake? It should be latter.

Reply:
my bad, the habit of coding in javascript/c#. I've fixed it